### ISSUE TITLE
Installation doesn't work.

### ISSUE THREAD

**Issue by cwells** [boxeehacks/boxeehack#495]
- Followed instructions in README.
- Red light comes on for a while, then reboots.
- Go into Settings->Network->Servers, change hostname to  "boxeebox; sh /data/hack/boot.sh", re-enable windows sharing.

Nothing is changed. Reboot again, still nothing (aside from having to go through the network wizard to reconnect Boxee to the internet). 

Boxee firmware is 1.5.1.23735-b6a5be1.

Any suggestions?

**Comment by cwells** [comment]
I suspected there might be a network issue, so I also tried adding the /hack directory to the USB stick. No luck. I've also tried reformatting the USB to FAT32 using mtools and gparted, but nothing changes. The only new "feature" is that I have to manually enable the network after each reboot. :-P

**Comment by jampeef** [comment]
did you name the usb-stick BOXEE

**Comment by ghost** [comment]
Format a USB stick or SD card and name it 'BOXEE' (be sure to use only capitals)
Copy the contents of the 'install' folder that you downloaded earlier to the USB stick or SD card
Plug the USB stick or SD card into your Boxee Box
Go into 'Settings'->'Network'->'Servers'
Enable 'Enable Windows file sharing'
Add ';sh[space]/media/BOXEE/install.sh' to your 'Host Name'. After you're done it should look like 'boxeebox;sh /media/BOXEE/install.sh'

name the stick BOXEE, and instert a space between sh and path.
you should only copy the content of install to the stick!

**Comment by igolfchip** [comment]
I am having the same exact issues installing as @cwells.
I HAVE reformatted the USB to Fat32
There is NOTHING else on the drive except the 4 files
I correctly typed in the string into the boxee servers page
My boxee does go red
it reboots and no changes to the system
the server hostname has been changed to: "sh /data/hack/boot.sh"

Please advise on how to correct this issue?

**Comment by cwells** [comment]
@scaliebe Yes, did all that (otherwise I wouldn't have gotten the red light on the Box).

**Comment by cwells** [comment]
I also notice that Boxee seems to crash a lot more frequently now, even though I don't have the hack enabled. Also every time it reboots I have to run the network wizard. Will running the uninstall script return it to its previous state?

**Comment by datakneus** [comment]
I have the exact same problem as cwells and igolfchip. Help would be appreciated!

**Comment by jampeef** [comment]
Just try it again, it happened to me once that it didnt install, might be coz of internet connection or something as it check the git for newest version on install..
or try and powercycle the boxee.

**Comment by Web-Wench** [comment]
I'm having the same problem. I've followed the instructions to the letter. I've tried installing 3 times. I get the same results as igolfchip. Anyone have any insight into this?

**Comment by Web-Wench** [comment]
Well apparently 4th time was a charm. Worked!

**Comment by cwells** [comment]
As a bonus, today my Boxee has started losing all its settings. I have to login with username/password and I have to re-setup all my sources (twice today).

Not trying to complain, but my Boxee has gone from being a piece of crap to being an unusable piece of crap. I didn't think Boxee could get worse, but I was clearly being pessimistic.

**Comment by cwells** [comment]
And now I'm screwed. I tried setting up my user account and my sources, and then doing a shutdown from the home screen, hoping that would get them saved before the next crash. Unfortunately, because the network doesn't come up (see my previous comments), I can't login. Because I can't login, I can't bring the network up. Catch-22.

I expect this thing is destined for Goodwill or the garbage.

**Comment by cwells** [comment]
I'd strongly recommend people think twice or three times before installing this hack.

**Comment by cwells** [comment]
It's also clear that the project admins here don't read the bug reports. Keep that in mind before you consider installing.

**Comment by cwells** [comment]
Restore factory settings:

http://forums.boxee.tv/showthread.php?t=22198

After that, the installation works. Perhaps this should be the recommended approach?

**Comment by jampeef** [comment]
@cwells well that should be if all else fail option, not recommended. most ppl wanna keep their settings

**Comment by cwells** [comment]
@jampeef It should at least be mentioned in the installation guide. Granted it was late at night and I was on my nth attempt, so my patience was undoubtedly already strained, but the fact that Boxee might have a factory reset didn't even occur to me until I was about ready to throw the damn thing into the garbage ;-)

**Comment by cwells** [comment]
I actually suspect that the Boxee settings were corrupt before I began the installation. I've had moderate stability issues with it for a few months and it seems much better now. Doing a factory reset once a year might be a good idea.

**Comment by mummelumme** [comment]
Just factory restored boxee and tryi to install boxee hacks again. Got the same problem as @cwells but this link doens't work anymore :( http://forums.boxee.tv/showthread.php?t=22198
Anybody there can tell me what it link to?

**Comment by pianoman235** [comment]
I have same problem lil help

**Comment by powval** [comment]
I am having the same exact issues installing as @cwells.
I HAVE reformatted the USB to Fat32
There is NOTHING else on the drive except the 4 files
I correctly typed in the string into the boxee servers page
My boxee does go red
it reboots and no changes to the system
the server hostname has been changed to: "sh /data/hack/boot.sh" I've read ome were I have to use network go into servers to activate the install but can't seee anything related to that any advice please thanks

**Comment by Robbie711** [comment]
try the following.
boxeebox; sh /media/BOXEE/install.sh

Look here:
http://logiclounge.com/2013/09/27/make-boxee-box-useful/

On Saturday 2/20/2016, [powval] said:

---

I am having the same exact issues installing as 
https://github.com/cwells@cwells.
I HAVE reformatted the USB to Fat32
There is NOTHING else on the drive except the 4 files
I correctly typed in the string into the boxee servers page
My boxee does go red
it reboots and no changes to the system
the server hostname has been changed to: "sh /data/hack/boot.sh" I've 
read ome were I have to use network go into servers to activate the 
install but can't seee anything related to that any advice please thanks

­
Reply to this email directly or 
https://github.com/boxeehacks/boxeehack/issues/495#issuecomment-186671317view 
it on GitHub.

**Comment by powval** [comment]
Thanks’ for replying

I did get it work finally I think it worked (crazy) because I discarded the
hand remote and give the orders to the onscreen order thanks again

Tony

**Comment by MMMcMahon** [comment]
For anyone stumbling across this issue while trying to fix - the suggestion at http://boxeed.in/forums/viewtopic.php?f=5&t=1454 helped me.   Specifically, delete your existing download folder via changing the hostname to `boxeebox; rm -fr /download/*`.  After that, re-attempt to install the hack via `boxeebox:sh /media/BOXEE/install.sh`.  Yay!

**Comment by Alfarovan** [comment]
MMMMcMahon you are a genius!!!! I tried EVERYTHING to make this work.

**Comment by gl01bxg2003** [comment]
Is there anyone who still has the files for Boxee Hacks version 1.5.3 . I tried everything, I simply can't find them.
